### PR TITLE
TWON-16900-KernelPackageUnified

### PR DESCRIPTION
**Pull request by lluisdiaz**

### CODE REVIEW

**Comment by lluisdiaz** [comment]
Ahora si, tested :)

**Review by faragon2n — DISMISSED** [review]
LGTM

File: make_var_mx6ul_dart_debian.sh [hunk 1]
```diff
@@ -768,15 +768,43 @@ function make_kernel_modules() {
...
+###################### Twonav kernel package fix for older kernel  packages ##########
+function pkg_info_gen() {
+	local OLD_KERN_BRANDS="twonav os"
+	local OLD_KERN_MODELS="trail aventura"
+	local BREAK_VERSION="1.0.19"
+	echo -n "$1"
+	local i j cnt=0
+	for i in $OLD_KERN_BRANDS ; do for j in $OLD_KERN_MODELS ; do
+		if ((cnt++ > 0)) ; then echo "," ; fi
+		echo -en "\t$2-4.1.15-$i-$j-2018 (<= $BREAK_VERSION)"
```
Comment (faragon2n) [inline make_var_mx6ul_dart_debian.sh:780]: @lluisdiaz may be the 4.1.15 and the 2018 may be unhardwired (?), also the BREAK_VERSION could be moved as a global variable (?)

**Review by faragon2n — CHANGES_REQUESTED** [review]
LGTM, just a minor appointment

File: make_var_mx6ul_dart_debian.sh (hunk 1 above)
Comment (lluisdiaz) [reply make_var_mx6ul_dart_debian.sh:780]: - why to move BREAK_VERSION to a global? this is some kind of patch and this way is localized.
- this debian script is based on 4.1.15 kernel version and our way to handle this devices is with 2018, i see no porblem with that.. or I don't understand the question :)

**Review by faragon2n — APPROVED** [review]
LGTM :-)

**Review by dnietotwonav — APPROVED** [review]

**Review by tpaschidis — APPROVED** [review]

**Pull request merged**